src/test/java/com/liferay/blade/cli/CreateCommandTest.java

```diff
@@ -1331,6 +1338,32 @@ public void testCreateWorkspaceTypeValid() throws Exception {
 		Assert.assertEquals(1, StringUtils.countMatches(content, '}'));
 	}
 
+	@Test
+	public void testLiferayVersion() throws Exception {
```

Comment:
Lets not use mocks for this but real integration tests where really invoke a template and assert that we got the 7.1 templates.